.travis.yml

```diff
@@ -7,6 +7,12 @@ cache:
   directories:
     - node_modules
 before_install:
+  - echo 'setting up locales...'
+  - sudo apt-get update
+  - sudo apt-get install language-pack-tr
+  - sudo /etc/init.d/postgresql stop
```

Comment:
is it necessary to restart postgresql? asking

Reply:
yep, apparently so. There are a few places it mentions this - but see here:

https://docs.travis-ci.com/user/database-setup/#PostgreSQL-and-Locales